docs/runtime/runtime-metadata.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "The Metadata of Runtime"
+---
+
+In blockchain, the logic to process transactions and modify storage is usually called STF (State Transition Function). The more popular name among Substrate developers is runtime. 
+
+Substrate module-based metadata provides an overview of blockchain runtime. It helps external code introspect modules, storage items, dispatchable functions and events.
+
+## SRML Metadata
+
+**[Srml-metadata](https://github.com/paritytech/substrate/tree/master/srml/metadata)** is not a "well-known" component of [SRML](runtime/substrate-runtime-module-library.md), since it doesn't provide additional storage entries, dispatchable functions and events. 
```

Comment:
We should avoid statements like this. We have no idea how many people understand srml-metadata or at what level.

Reply:
This statement is not based on the user's understanding, but comes after this [page](https://substrate.dev/docs/en/next/runtime/substrate-runtime-module-library)

Reply:
> but comes after this page.

So?

That people don't understand it is your assumption and the reason you're writing the page. You don't need to state that.